tools/topology/topology2/include/dais/dmic.conf

```diff
@@ -0,0 +1,113 @@
+#
+# Intel DMIC DAI
+#
+# All attributes defined herein are namespaced by alsatplg to "dmic.attribute_name"
+#
+# Usage: this component can be used by declaring in the "dais" field of
+# a parent object. i.e.
+#
+# For Capture
+# Object.DMIC."M.capture" {
+#	name	"dmic16k"
+#	id	2
+#	Object.Base.hw_config."0" {
+#		id  0
+#	}
+#	Object.Base.pdm_config."0" {
+#		ctrl_id 0
+#	}
+# }
+#
+#
+# Where M is DAI index in the firmware
+
+Class.Dai."DMIC" {
+
+	#
+	# Argument used to construct DAI widget
+	#
+	# DAI Index
+	DefineAttribute."dai_index" {
+		token_ref	"sof_tkn_dai.word"
+	}
+
+	DefineAttribute."direction" {
+		type "string"
+	}
+
+	DefineAttribute."dai_type" {
+		type	"string"
+		token_ref	"sof_tkn_dai.string"
+	}
+
+	# Backend DAI Link ID matching with the machine driver
+	DefineAttribute.id {}
+
+	DefineAttribute.default_hw_config_id {}
+
+	DefineAttribute.name {
+		type	"string"
+	}
+
+	DefineAttribute.driver_version {
+		token_ref	"sof_tkn_dmic.word"
+	}
+
+	DefineAttribute.clk_min {
+		token_ref	"sof_tkn_dmic.word"
+	}
+
+	DefineAttribute.clk_max {
+		token_ref	"sof_tkn_dmic.word"
+	}
+
+	DefineAttribute.duty_min {
+		token_ref	"sof_tkn_dmic.word"
+	}
+
+	DefineAttribute.duty_max {
+		token_ref	"sof_tkn_dmic.word"
+	}
+
+	DefineAttribute.num_pdm_active {
+		token_ref	"sof_tkn_dmic.word"
```

Comment:
@juimonen  Just find that some used tokens are not defined, `sof_tkn_dmic`,  `sof_tkn_dmic_pdm`. please define them in common/tokens.conf

Reply:
ok.